.github/workflows/push.yml

```diff
@@ -0,0 +1,43 @@
+name: GitHub Actions
+on: [push]
+env:
+  APP_IMAGE: keboola-ex-storage
+  KBC_DEVELOPERPORTAL_APP: "keboola.ex-storage"
+  KBC_DEVELOPERPORTAL_VENDOR: "keboola"
+  KBC_DEVELOPERPORTAL_USERNAME: "keboola+ex_storage_gh_actions"
+  KBC_DEVELOPERPORTAL_PASSWORD: ${{ secrets.KBC_DEVELOPERPORTAL_PASSWORD }}
+  # https://connection.keboola.com/admin/projects/2703/extractors/keboola.ex-storage/764729155
+  KBC_APP_TEST_CONFIG_ID: 764729155
+  KBC_TEST_BUCKET: in.c-ex-storage
+  KBC_TEST_URL: https://connection.keboola.com/
+  KBC_TEST_TOKEN: ${{ secrets.KBC_TEST_TOKEN }}
+  KBC_TEST_WRITE_TOKEN: ${{ secrets.KBC_TEST_WRITE_TOKEN }}
+  KBC_STORAGE_TOKEN: ${{ secrets.KBC_STORAGE_TOKEN }}
+jobs:
+  Build:
```

Comment:
IMHO by to bolo dobre dat podla sablony, ... je tam GithubAction na `push` aj `deploy`, mame to uz oskusane na viacerych komponentach.

https://github.com/keboola/component-generator/blob/master/templates-ci/php-component/github-actions/.github/workflows/push.yml